config/parity_check_endpoints.yml

```diff
@@ -1,7 +1,66 @@
 get:
+  "/api/v1/npq-applications":
+    paginate: true
+  "/api/v2/npq-applications":
+    paginate: true
+  "/api/v3/npq-applications":
+    paginate: true
+
+  "/api/v1/npq-applications/:id":
+    id: 'lead_provider.applications.order("RANDOM()").limit(1).pick(:ecf_id)'
```

Comment:
possible feature, option to run the single request multiple times? eg: "run_count: 100"

Reply:
Yeah one for the future maybe as this would have knock on effects to how we display/store the results

Reply:
Hmm, i'm thinking we could do it now for free by duplicating the same URL like this:
- `/api/v1/npq-applications/:id#1`
- `/api/v1/npq-applications/:id#2`
- `/api/v1/npq-applications/:id#3`

lol